keps/sig-node/2371-cri-pod-container-stats/README.md

```diff
@@ -286,8 +287,7 @@ as cAdvisor is fine tuned to perform in an adequate manner.
 ### Stats Summary API
 
 #### CRI Implementation
-The CRI implementation will need to be extended to support reporting the full set of container-level from the [Summary API](#summary-container-stats-object).
-
+The CRI implementation will need to be extended to support reporting the full set of container-level from the [Summary API](#summary-container-stats-object). A new gRPC call will also be added to the CRI that allows reporting for metrics currently exported by cAdvisor, but are outside the scope of the Summary API. This new gRPC call will return a Prometheus metric based response which Kubelet can export. Additionally, `PodAndContainerStatsFromCRI` feature gate support will be added to only report Prometheus based metrics from the CRI when calling `/metrics/cadvisor` endpoint when the feature gate is enabled. The additional metrics we support will need to be added to the individual container runtimes.
```

Comment:
From previous release, the PRR Questionnaire section for "Does enabling the feature change any default behavior?" said "Enabling this behavior means some stats endpoints will not be filled: some entries in /metrics/cadvisor"

Is this still accurate with this change? Is there anything about the structure of the metrics or what metrics are available that should be included in that PRR section?

Reply:
yeah this is still true. accelerator metrics, for instance, are being dropped. The table at the top has a column dedicated to saying whether we're aiming to support them